ntrans.py

```diff
@@ -71,7 +72,7 @@ def read_ngram_files(user_choices: Dict[str, Any]) -> None:
                     break
                 ngrams[n].append(row[0])
 
-    machine_translate_ngrams(ngrams, user_choices)
+    machine_translate_ngrams(ngrams, user_choices, progress_queue)
 
```

Comment:
Hmm, I didn't realize ntrans.py got added in this PR as well. I started a new branch from generate_progress branch, but never did git add ntrans.py (if I didn't do it by mistake without noticing)

Reply:
Run `git diff --cached` before committing, and use `git log --all --pretty --oneline` when working with branches :)

Reply:
Yeah, I ran --staged, but didn't look through it well enough. This happens because I created this new branch from the already existing non-main branch. I wanted to have all methods etc created in generate_progress branch.. Can you help me understand how I can do that without committing the generate_progress changes ?

When are you coming on hexchat today?

Reply:
See my git guide: https://akuli.github.io/git-guide/pr.html#making-another-pr

In short, you need to go to main branch and pull. Because this is your own repo and not a fork, the pulling is slightly simpler than in git-guide: just `git pull` instead of `git pull https://github.com/user/repo`.